Tests/AttributedStringStyleTests.swift

```diff
@@ -555,6 +555,134 @@ class StringStyleTests: XCTestCase {
         XCTAssertFalse(stillHasAltSixDict)
     }
 
+    func testStyleBlockRules() {
+        let string = "0<one attr1=\"11\">1<two attr2=\"12\">2</two></one>"
```

Comment:
[Raw strings](https://www.hackingwithswift.com/articles/162/how-to-use-raw-strings-in-swift) make this a lot nicer:

```suggestion
        let string = #"0<one attr1="11">1<two attr2="12">2</two></one>"#
```

Reply:
Also, I find the use of "11" and "12" in the mix to be confusing. I'd love longer and/or less ambiguous names, just to make it easier to read this code 🙂

Reply:
In this test, I've tried not only to extract and assert attribute values but also to have them included in the styling work.

So I've chosen one NSAttributeString parameter (`baselineOffset`) and tried to produce a style with a `baselineOffset` parsed from attribute's value in each of `styleBlock`s. Because of that, values of these attributes are numbers.

It doesn't matter for me what NSAttributedString style to use in the test, maybe there is a better fit instead of `baselineOffset`. E.g. we can have font names as attribute values and try to construct and return `.font()` style within blocks...